apps/extension/src/features/accounts/account-card.tsx

```diff
@@ -0,0 +1,376 @@
+  // style the events
+  const fullConfig = resolveConfig({
+    ...tailwindConfig,
+    content: ["./src/**/*.{html,js,ts,tsx}"],
+  });
+  console.log(fullConfig);
+
+  const colors = Object.keys(RadixColor) as RadixColor[];
+  const [container, setContainer] = React.useState<HTMLDivElement | null>(null);
+
+  const themeColors = fullConfig.theme?.colors as Record<RadixColor, string>;
+  const utils = trpcApi.useContext();
+  const { mutateAsync: updateEventColor } =
+    trpcApi.account.updateColor.useMutation({
+      onSuccess() {
+        utils.user.myAccounts.invalidate();
+      },
+    });
+
+  return (
+    <Tooltip.Provider>
+      <div className="flex justify-between py-6 pr-6 space-x-4 border rounded-lg h-30 border-mauve-6 pl-9">
+        {/* Info */}
+        <div className="flex w-full gap-4">
+          {/* Avatar */}
+          <div
+            className="self-center w-10 h-10 leading-10 text-center rounded-full"
+            style={{
+              backgroundColor: themeColors?.[account.eventColor][7],
+              color: themeColors?.[account.eventColor][11],
+            }}
+          >
+            {initials}
+          </div>
+          {/* the account color for the given account */}
+          <div className="flex flex-col justify-center">
+            <div className="text-sm font-medium">{account.eventColor}</div>
+          </div>
+          {/* Email, Colors & Badges */}
+          {/* <div className="flex flex-grow"> */}
+          <div className="flex flex-col items-start flex-grow px-10 ">
+            {/* this div makes sure that the children have the same width as determined by colors */}
+            <div>
+              <div className="flex justify-between">
+                {/* a div displaying the email on the left side and a conditional organizer badge on the right */}
+                <div className="text-sm font-normal">{account.email}</div>
+                {account.isPrimary && (
+                  <Tooltip.Root delayDuration={300}>
+                    <Tooltip.Trigger asChild>
+                      <div className="px-2 py-1 text-xs text-white rounded bg-blue-9 cursor-help">
+                        Organizer
+                      </div>
+                    </Tooltip.Trigger>
+                    <Tooltip.Content
+                      align="end"
+                      side="right"
+                      sideOffset={10}
+                      className="z-10"
+                    >
+                      <div className="px-5 py-6 bg-white rounded-lg shadow-2xl w-[350]">
+                        <div className="font-semibold">Organizer Account</div>
+                        <div className="pt-2">
+                          The selected account will be the organizer for all
+                          events created.
+                        </div>
+                      </div>
+                    </Tooltip.Content>
+                  </Tooltip.Root>
+                )}
+              </div>
```

Comment:
fix height on outer div would remove layout shift here

![CleanShot 2022-11-23 at 00 22 07](https://user-images.githubusercontent.com/51714798/203440889-1053a580-a57c-44e6-a136-c99611c1750e.gif)

Reply:
fixed by defining a bigger line-height on the email text
https://github.com/agreeto/agreeto/pull/11/commits/49566c6ba9d5e17a74186bd37a59680af5b071eb